README.md

```diff
@@ -45,12 +45,12 @@ $ bin/setup
 $ rails s
 ```
 
-Then navigate to `http://localhost:3000` in your browser to view the app. Two user accounts will be available for logging in: **User** and **Admin**. Use one of the logins to access the app:
- * User
-   * Email: `user@example.com`
```

Comment:
While I like that you've updated documentation for admin accounts, it seems that you've removed from the documentation the simple user account :$